pkg/apis/okctl.io/v1alpha1/cluster_v1alpha1.go

```diff
@@ -243,11 +247,12 @@ func NewDefaultCluster(name, env, org, repo, team, accountID string) Cluster {
 			HighAvailability: true,
 		},
 		Integrations: &ClusterIntegrations{
-			ALBIngressController: true,
-			ExternalDNS:          true,
-			ExternalSecrets:      true,
-			Cognito:              true,
-			ArgoCD:               true,
+			ALBIngressController:      false,
+			AWSLoadBalancerController: true,
```

Comment:
Maybe we should get rid of the ALBIngressController? (Not in this PR) Or is it some kind of difference in functionality that we need to keep around

Reply:
Nope, we should remove it, just figured I would do it in a separate PR. 👍🏽